runtests.sh

```diff
@@ -27,13 +27,15 @@ catch() {
 
 install_glassfish() {
   mvn clean -N org.apache.maven.plugins:maven-dependency-plugin:3.2.0:copy \
+  -Pstaging \
   -Dartifact=org.glassfish.main.distributions:glassfish:${GF_VERSION}:zip \
   -Dmdep.stripVersion=true \
   -DoutputDirectory=${WORKSPACE}/bundles
 }
 
 install_jacoco() {
   mvn -N org.apache.maven.plugins:maven-dependency-plugin:3.2.0:copy \
+  -Pstaging \
```

Comment:
This is redundant, jacoco is not in staging, just in central, also this command doesn't use poms  ;)